src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PreviewPrintController.Windows.cs

```diff
@@ -84,7 +84,7 @@ public override Graphics OnStartPage(PrintDocument document, PrintPageEventArgs
                 _graphics.TranslateTransform(document.DefaultPageSettings.Margins.Left, document.DefaultPageSettings.Margins.Top);
             }
 
-            _graphics.PrintingHelper = printGraphics;
+            _graphics!.PrintingHelper = printGraphics;
```

Comment:
Maybe we should remove null check on row 72, redundant one, then no need to bang

Reply:
Agreed, doesn't make any sense to keep here. Same with DefaultPrintController